meta-genivi-dev/recipes-multimedia/audiomanager/audiomanager-demo_1.2.bb

```diff
@@ -6,11 +6,9 @@ LIC_FILES_CHKSUM = "file://AudioManagerPoC/README;md5=be0fe05c89ff7a2632eb080f72
 
 DEPENDS = "qtbase qtdeclarative qtwebkit"
 
-BRANCH = "Intreprid_stable_branch"
-VERSION = "6.1"
-
+SRCREV = "daf851ee7a41d1b0572c0c95e15f61e427ce97f1"
```

Comment:
One more thing: This is a separate change, hence should be in a separate commit.